plugin/pkg/auth/authenticator/token/oidc/oidc_test.go

```diff
@@ -391,5 +391,6 @@ func TestOIDCAuthentication(t *testing.T) {
 		if !reflect.DeepEqual(tt.userInfo, user) {
 			t.Errorf("#%d: Expecting: %v, but got: %v", i, tt.userInfo, user)
 		}
+		client.Close()
```

Comment:
@gmarek You are right. Let me replace the close with sending struct{}{}, as the channel is an unbuffered channel, sending to that channel happens after the channel is received, in which case this should guarantee there is not flying requests then.